products/models.py

```diff
@@ -0,0 +1,71 @@
+from django.db import models
+
+
+class Category(models.Model):
+    name = models.CharField(max_length=40, unique=True)
+    is_active = models.BooleanField(default=True)
+    creation_date = models.DateTimeField(auto_now_add=True)
+
```

Comment:
Search for tools that can provide TimeStamps and activation features instead of adding it by your self.